R/api_cache.R

```diff
@@ -0,0 +1,98 @@
+#' @title Data Caching API
+#' @description Functions to manage cache data across sessions.
+#' @param path character, path to cache directory, Default: cache_dir()
+#' @param \dots arguments passed to [list.files][base::list.files]
+#' @param verbose logical, show message on success of [cache_destroy][cache_destroy]
```

Comment:
```suggestion
#' @param verbose logical, show message on success of [cache_destroy()]
```
similar for the other places